packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -446,5 +452,675 @@ class _CupertinoTextSelectionControls extends TextSelectionControls {
   }
 }
 
+// Renders the content of the selection menu and maintains the page state.
+class _CupertinoTextSelectionToolbarContent extends StatefulWidget {
+  const _CupertinoTextSelectionToolbarContent({
+    Key key,
+    @required this.children,
+    @required this.isArrowPointingDown,
+  }) : assert(children != null),
+       assert(children.length > 0), // ignore: prefer_is_empty
```

Comment:
Is this to get around the const?

Reply:
Yes, `.isNotEmpty` isn't const.  I'll add a comment about it.